### ISSUE TITLE
How to autoplay youtube video?

### ISSUE THREAD

**Issue by bsde1245** [ashleydw/lightbox#324]
So, you're experiencing a problem?

If you're not sure how to use the lightbox, or are experiencing a problem with it, please ask on [StackOverflow](http://stackoverflow.com/) - you will most definitely get a faster response time and it leaves this GitHub issues for actual bugs and not support questions.

If you really do think you have a bug, please:

- [http://ashleydw.github.io/lightbox/#videos ] Provide a link to your site or a full example

- [Chrome, windows 10 ] Browser name and version

In your site also autoplay of youtube not working. 

http://ashleydw.github.io/lightbox/#videos

**Comment by mediageni** [comment]
yes, looking for solution for that too. autoplay=1 is not working.

**Comment by m1dst** [comment]
autoplay=1 on the link seems to work for me in Chrome OK.
